src/test/java/hudson/plugins/git/GitExceptionTest.java

```diff
@@ -54,24 +55,13 @@ public void throwsGitExceptionExpectedMessageWithCause() {
         throw new GitException(message, new IOException("Custom IOException message"));
     }
 
-    @Test
-    public void initDefaultImplThrowsGitException() throws GitAPIException, IOException, InterruptedException {
-        File badDirectory = new File("/this/is/a/bad/dir");
-        if (isWindows()) {
-            badDirectory = new File("\\\\badserver\\badshare\\bad\\dir");
-        }
-        GitClient defaultClient = Git.with(TaskListener.NULL, new EnvVars()).in(badDirectory).using("Default").getClient();
```

Comment:
This test was nonsense—it _did_ throw an exception (running as `root` or not), but only because it was trying to running a command named `Default`, which of course failed.